job_bundles/gsplat_pipeline/README.md

```diff
@@ -0,0 +1,513 @@
+show the download progress in your browser, or present a CLI command you can use to download.
+
+The .ply file will be saved to the location you selected when submitting the job. To view the result
+in your browser, open the [SuperSplat Editor](https://superspl.at/editor) website and drag/drop
+the file from your operating system's file browser onto the SuperSplat page. The initial view
+will look something like this:
+
+![supersplat initial view](.readme_images/supersplat_initial_view.png)
+
+After toggling the Show/Hide Splats option on the right, rotating the scene by about ninety degrees,
+and navigating around the scene, here's the subject of the capture:
+
+![supersplat found subject](.readme_images/supersplat_found_subject.png)
+
+With SuperSplat, you can edit your Gaussian Splatting. Here's the result of selecting a sphere,
+inverting the selection, and deleting those splats:
+
+![supersplat sphere cropped](.readme_images/supersplat_sphere_cropped.png)
+
+There are many tutorials and [documentation pages](https://github.com/playcanvas/supersplat/wiki) to learn how to use SuperSplat,
+or import your .ply file into your tool of choice.
+
+## Understand the cost of your training jobs
+
+Deadline Cloud monitor includes a usage explorer feature that can estimate the cost of the jobs you run
+and help you understand how much each training costs. For the sample, I trained at higher quality with a larger
+number of images than default and the Structure-from-Motion and training took about an hour. The example costs
+shown here are for an on-demand CUDA fleet. If you're comfortable running your tests on off-peak hours when
+CUDA-capable instances are available with low enough interruption rates, you could use a spot CUDA fleet
+at reduced cost. What is shown in usage explorer does not include costs outside of your job run time, such as idle worker
+instance time or storage on your S3 bucket.
+
+![Deadline Cloud monitor usage explorer by job](.readme_images/deadline_cloud_monitor_usage_explorer_by_job.png)
+
+## This sample is a starting point<a id='next-steps'></a>
+
+You can get good mileage out of the gsplat_pipeline job as it is, but we created it to be
+pulled apart, edited, and remixed.
+
+### Run the job anywhere with the Open Job Description CLI
+
+Following the spirit of the Open Job Description
+[Design Tenets](https://github.com/OpenJobDescription/openjd-specifications/wiki/Design-Tenets),
+the job template in this bundle is portable. You can run it locally using the [openjd CLI](https://github.com/OpenJobDescription/openjd-cli#readme)
+tool, installable into your Python via `pip install openjd-cli`. See the
+[Introduction to Creating a Job](https://github.com/OpenJobDescription/openjd-specifications/wiki/Introduction-to-Creating-a-Job)
+documentation for some ideas on local development setup.
+
+I created an EC2 instance using the [AWS deep Learning AMI with Conda](https://docs.aws.amazon.com/dlami/latest/devguide/overview-conda.html),
```

Comment:
Style nit: text changes to the first person here. Would it make sense to keep it consistent with second or third person?

Reply:
Yes, that's a good point. I'll do that and the other fix in a followup!